services/spar/test-integration/Util/Scim.hs

```diff
@@ -396,7 +396,9 @@ instance IsUser Scim.User.User where
     maybeHandle = Just . Just . Handle . Scim.User.userName
     maybeName = Just . fmap Name . Scim.User.displayName
     maybeTenant = const Nothing
-    maybeSubject = Just . fmap SAML.opaqueNameID . Scim.User.externalId
+    maybeSubject =
+        let emailNameID s = SAML.NameID (SAML.UNameIDEmail s) Nothing Nothing Nothing
+        in Just . fmap emailNameID . Scim.User.externalId
```

Comment:
Doesn't this give you 2 `Just`?  one from `externalId`, one Just?  I know it was like that before, so it's probably correct.  Haven't checked the context.  Please ignore.

Reply:
True, but it's supposed to be like that. I'll add comments and rewrite the class a bit.